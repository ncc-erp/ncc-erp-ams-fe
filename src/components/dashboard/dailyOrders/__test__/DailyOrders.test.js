import renderer from 'react-test-renderer';
import { DailyOrders } from '../';

test('Test DailyOrders component', () => {
  const component = renderer.create(
    <DailyOrders></DailyOrders>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});