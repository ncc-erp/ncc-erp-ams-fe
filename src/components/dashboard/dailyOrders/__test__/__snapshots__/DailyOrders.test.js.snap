// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test DailyOrders component 1`] = `
<div
  className="daily-order-wrapper"
>
  <div
    className="title-area"
  >
    <h3
      className="ant-typography"
      onClick={null}
      style={
        Object {
          "WebkitLineClamp": undefined,
        }
      }
    />
    <div
      className="title-area__number"
    >
      <span
        className="ant-typography"
        onClick={null}
        style={
          Object {
            "WebkitLineClamp": undefined,
          }
        }
      >
        <strong>
          0
           
        </strong>
      </span>
    </div>
  </div>
  <div>
    Content
  </div>
</div>
`;
